rmw_fastrtps_shared_cpp/src/rmw_event.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <unordered_set>
+
+#include "rmw/impl/cpp/macros.hpp"
+
+#include "rmw_fastrtps_shared_cpp/rmw_common.hpp"
+#include "types/event_types.hpp"
+
+static const std::unordered_set<rmw_event_type_t> __rmw_event_type_set{
```

Comment:
Don't use `__`, it is reserved for the standard library.
Use `g_` instead since this is a global.

Reply:
Done.